examples/cur/reportdefinition.yaml

```
@@ -0,0 +1,191 @@
+apiVersion: cur.aws.upbound.io/v1beta1
+kind: ReportDefinition
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: example_cur_report_definition
+  name: example-cur-report-definition
+spec:
+  forProvider:
+    additionalArtifacts:
+    - REDSHIFT
+    - QUICKSIGHT
+    additionalSchemaElements:
+    - RESOURCES
+    compression: GZIP
+    format: textORcsv
+    region: us-east-1
+    s3Bucket: test-definition-bucket
+    s3Region: us-east-1
+    timeUnit: HOURLY
+
+---
+
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/external-name: test-definition-bucket
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: b
+  name: test-definition-bucket
+spec:
+  forProvider:
+    forceDestroy: true
+    region: us-east-1
+
+---
+
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: BucketPolicy
+metadata:
+  name: example
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: example_cur_report_definition
+spec:
+  forProvider:
+    region: us-east-1
+    bucketSelector:
+      matchLabels:
+        testing.upbound.io/example-name: b
+    policy: |
+      {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "billingreports.amazonaws.com"
+                },
+                "Action": [
+                    "s3:GetBucketAcl",
+                    "s3:GetBucketPolicy"
+                ],
+                "Resource":"arn:aws:s3:::test-definition-bucket",
+                "Condition": {
+                    "StringEquals": {
+                        "aws:SourceArn": "arn:aws:cur:us-east-1:${data.aws_account_id}:definition/*",
+                        "aws:SourceAccount": "${data.aws_account_id}"
+                    }
+                }
+            },
+            {
+                "Sid": "Stmt1335892526596",
+                "Effect": "Allow",
+                "Principal": {
+                    "Service": "billingreports.amazonaws.com"
+                },
+                "Action": "s3:PutObject",
+                "Resource": "arn:aws:s3:::test-definition-bucket/*",
+                "Condition": {
+                    "StringEquals": {
+                        "aws:SourceArn": "arn:aws:cur:us-east-1:${data.aws_account_id}:definition/*",
+                        "aws:SourceAccount": "${data.aws_account_id}"
+                    }
+                }
+            }
+          ]
+        }
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: r
```

nitpick: any reason for not being consistent? 

```suggestion
    testing.upbound.io/example-name: example
```